src/test/kotlin/net/fabricmc/loom/test/kotlin/KotlinClassMetadataRemappingAnnotationVisitorTest.kt

```diff
@@ -0,0 +1,12 @@
+package net.fabricmc.loom.test.kotlin
+
+import org.junit.jupiter.api.Test
+
+// See: https://github.com/JetBrains/kotlin/blob/master/libraries/kotlinx-metadata/jvm/test/kotlinx/metadata/test/MetadataSmokeTest.kt#L67
+class KotlinClassMetadataRemappingAnnotationVisitorTest {
+
+    @Test
+    fun simpleTest() {
+
+    }
```

Comment:
Does this actually do anything?

Reply:
No I was going to write unit tests for it and clearly got bored. 😂